src/content/docs/features/timers.md

```diff
@@ -2,6 +2,12 @@
 title: Timers
 ---
 
-:::note
-This page is still under construction. If you'd like details on how to handle events, please ask us directly on [discord](/discord/).
-:::
+Membrane [programs](/concepts/programs/) can be invoked manually or automatically, on a timer. Membrane supports three types of timers:
+
+1. Delays
+2. Scheduled
+3. Crons
+
+To set up a timer, navigate to the Explorer panel, click on any program, find the [action](/concepts/the-graph#actions) you want to run, and click the clock icon to configure when the program should run.
+
+<!-- TODO: add video of creating a timer from the explorer -->
```

Comment:
I didn't add one yet because of the bug [mentioned on Discord](https://discord.com/channels/902671998902206475/1212831343814381578/1249851355732901980).

Reply:
We should also mention `$invokeIn`, `$invokeAt`, and `$cron` in this section.